omnibus/cookbooks/omnibus-supermarket/attributes/default.rb

```diff
@@ -332,7 +332,8 @@
 default['supermarket']['protocol'] = node['supermarket']['nginx']['force_ssl'] ? 'https' : 'http'
 default['supermarket']['pubsubhubbub_callback_url'] = nil
 default['supermarket']['pubsubhubbub_secret'] = nil
-default['supermarket']['redis_url'] = "redis://#{node['supermarket']['redis']['bind']}:#{node['supermarket']['redis']['port']}/0/supermarket"
+default['supermarket']['redis_url'] = 'redis://127.0.0.1:16379/0/supermarket'
```

Comment:
Took the opportunity to update this one attribute related to the change to remove the interpolation of default values. These interpolations are not evaluated later to pick up overrides which is what many humans reading these attributes think happens.

I'll also note here that I've observed that the `/supermarket` portion of the Redis connection URL does nothing; keys go into the numbered database at the "root" not under a "supermarket" namespace. And [the scheme for a Redis URI](https://www.iana.org/assignments/uri-schemes/prov/redis) has the path ending in a sole, lonely positive integer to be followed by optional parameters.

Some other time, I'll pull the `/supermarket` portions out of the defaults. I did not include that in this change to minimize differences.